.github/workflows/release-jetbrains.yml

```diff
@@ -0,0 +1,80 @@
+name: release-jetbrains
```

Comment:
The workflow originally came from https://github.com/JetBrains/intellij-platform-plugin-template/blob/main/.github/workflows/release.yml